index.html

```diff
@@ -0,0 +1,172 @@
+                                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam modi repellendus nesciunt quasi corrupti.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class='bx bx-palette'></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    CC Ignites
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam modi repellendus nesciunt quasi corrupti.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">JOIN CC GLOBAL SUMMIT 2021</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="speakers">
+                    <h1 class="speakers-heading">Featured Speakers</h1>
+                    <ul class="speakers-list" id="speakers-list">
+                    </ul>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container bg-dark">
+                <div class="partner">
+                    <h1 class="partner-heading">Partner</h1>
+                    <ul class="partners-list">
+                        <li class="partner-list-item">
+                            <i class='bx bxl-firefox' ></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class='bx bxl-google' ></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class='bx bxl-airbnb'></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class='bx bxl-amazon' ></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class='bx bxl-yahoo' ></i>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="container">
+            <div class="footer">
+                <div class="logo">
+                    <a href="#">Logo</a>
+                </div>
+                <p class="rights">
+                    2021 Creative Commons Korea Some Rights Reserved
+                </p>
+            </div>
+        </div>
```

Comment:
Add another div below this one for the bottom footer
![right4](https://user-images.githubusercontent.com/58376020/123693602-8da42a80-d858-11eb-9ccb-e08fb012f379.png)

Reply:
This change has been made well.